UICatalog/UICatalog.cs

```diff
@@ -270,6 +272,21 @@ public UICatalogTopLevel ()
 				ScenarioListView.SelectedItem = _cachedScenarioIndex;
 			}
 
+			private void SetTitlePaddingToAlignScenariosHeader ()
+			{ 
+				// Pos.Anchor is internal so we have to use reflection to access it
```

Comment:
> What if SplitContainer did full borders (like FrameView) for each of it's panes.

I like this idea but I think it must be optional.  

There are cases where you don't want to loose the vertical space taken up by the border (e.g. in FileDialog2) or where it would be unwelcome to always have borders (e.g. when nesting `SplitContainer`s).